src/ui/public/filter_bar/lib/map_geo_bounding_box.js

```diff
@@ -1,20 +1,33 @@
 import _ from 'lodash';
+import { SavedObjectNotFound } from '../../errors';
 
 export function FilterBarLibMapGeoBoundingBoxProvider(Promise, courier) {
   return function (filter) {
     if (filter.geo_bounding_box) {
-      return courier
-      .indexPatterns
-      .get(filter.meta.index).then(function (indexPattern) {
+      function getParams(indexPattern) {
         const type = 'geo_bounding_box';
         const key = _.keys(filter.geo_bounding_box)
           .filter(key => key !== 'ignore_unmapped')[0];
-        const field = indexPattern.fields.byName[key];
         const params = filter.geo_bounding_box[key];
-        const topLeft = field.format.convert(params.top_left);
-        const bottomRight = field.format.convert(params.bottom_right);
+        const topLeft = indexPattern
+        ? indexPattern.fields.byName[key].format.convert(params.top_left)
+        : JSON.stringify(params.top_left);
+        const bottomRight = indexPattern
+         ? indexPattern.fields.byName[key].format.convert(params.bottom_right)
+         : JSON.stringify(params.bottom_right);
         const value = topLeft + ' to ' + bottomRight;
         return { type, key, value, params };
+      }
+
+      return courier
+      .indexPatterns
+      .get(filter.meta.index)
+      .then(getParams)
+      .catch((error) => {
+        if (error instanceof SavedObjectNotFound) {
+          return getParams();
+        }
+        throw error;
```

Comment:
What do we gain by throwing the error here?

Reply:
I want the promise to reject in order to preserve the old behavior for errors that are not of type `SavedObjectNotFound`.